deps/wrapper.cc

```diff
@@ -89,9 +91,14 @@ void initialize_coreworker_worker(
           (*returns)[0].second = std::make_shared<RayObject>(memory_buffer, nullptr, std::vector<rpc::ObjectReference>());
           return Status::OK();
         };
+    std::cerr << "Initializing julia worker coreworker" << std::endl;
     CoreWorkerProcess::Initialize(options);
 
+    std::cerr << "Starting julia worker task execution loop" << std::endl;
     CoreWorkerProcess::RunTaskExecutionLoop();
+
+    // doesn't seem to be reached...
```

Comment:
do we need to include if it doesn't then?

Reply:
this was an act of desperation. I'll clean these up (or make them ray debug logs or something)